sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/concatenate_data/concatenate_data_nodelet.hpp

```diff
@@ -68,9 +68,9 @@
 #include <tier4_autoware_utils/ros/debug_publisher.hpp>
 #include <tier4_autoware_utils/system/stop_watch.hpp>
 
-#include <autoware_auto_vehicle_msgs/msg/velocity_report.hpp>
```

Comment:
I think you can also remove this line, since this was the last part where pointcloud_preprocessor depended on autoware_auto_vehicle_msgs
https://github.com/autowarefoundation/autoware.universe/blob/125c648cf2b9e85953a4a5a8b774c8112e4335e5/sensing/pointcloud_preprocessor/package.xml#L20

Reply:
Thanks! Fixed at https://github.com/autowarefoundation/autoware.universe/pull/3877/commits/b92b1d1c2d30ef9d5846e00b948f2735f9860f4e